src/main/kotlin/Main.kt

```diff
@@ -76,3 +85,32 @@ fun scenario(numberOfCars: Int, capacity: Int) {
         }
     }
 }
+fun ListOfInterestInteger(ListOfInterest:MutableList<Boolean>):MutableList<Int>{
```

Comment:
I don't think that this function is necessary because we could do everything in Boolean (see below)